### ISSUE TITLE
Python Scraper FR still fails

### ISSUE THREAD

**Issue by zdavatz** [openZH/covid_19#255]
cc @baryluk 

```
Traceback (most recent call last):
  File "././scrape_fr.sh", line 49, in <module>
    assert m, "Can't find table"
AssertionError: Can't find table
FR -                      -       - FAILED 2020-03-27T09:23:46+01:00 Missing date. Missing cases https://www.fr.ch/covid19/sante/covid-19/coronavirus-statistiques-evolution-de-la-situation-dans-le-canton
```

**Comment by metaodi** [comment]
@zdavatz I created a fix in #249, can you check if that work for you?

**Comment by zdavatz** [comment]
@metaodi yes, thank you this works!
